.github/workflows/build-images.yml

```diff
@@ -0,0 +1,69 @@
+name: Building Docker Images
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  opensuse-leap-15-1:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          path: salt-toaster
+      - name: Checkout tools repo
+        uses: actions/checkout@v2
+        with:
+          repository: opensuse/salt
+          path: salt
+      - name: Install dependencies
+        working-directory: salt-toaster
+        run: pip install -r images/requirements.txt
+      - name: Build Image
+        working-directory: salt-toaster
+        env:
+          DISTRO: opensuse151
+          FLAVOR: devel
+        run: make build_image SALT_REPO="$GITHUB_WORKSPACE/salt"
+      - name: Login to registry
```

Comment:
There are also actions we could use for this as described in https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions

Reply:
Okay interesting. On one of the other pages they don't mention this. It is probably more reasonable to use the actions then our own things. I will refactor it later.